site/content/contribute/more-info/server/tests.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Tests"
+heading: "Server test guidelines"
+description: "Guidelines to write golang server tests"
+date: 2025-02-05T18:00:00-04:00
+weight: 2
+aliases:
+  - /contribute/server/tests
+---
+
+## Handling Flaky Tests
+
+A flaky test is one that exhibits both passing and failing results when run multiple times without any code changes. When our automation detects a flaky test on your PR:
+
+1. **Check if the Test is Newly Introduced**
+   - Review your PR changes to determine if the flaky test was introduced by your changes
+   - If the test is new, fix the flakiness in your PR before merging
+
+2. **For Existing Flaky Tests**
+   - Create a JIRA ticket titled "Flaky Test: {TestName}", e.g. "Flaky Test: TestGetMattermostLog"
+   - Copy the test failure message into the JIRA ticket description
+   - Add the `flaky-test` and `triage-global` labels
```

Comment:
I've created the flaky-test` lable to track flaky tests. That is the only process change this PR proposes.

Reply:
Thank you, sounds good.

So, who owns reviewing this label weekly?

Reply:
I suggest the ticket follow the regular triage process. @amyblais is usually one the one who redirects tickets from `triage-global` to the corresponding teams, right?